packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -76,7 +100,7 @@ export default class QuanticResultQuickview extends LightningElement {
       result: this.result,
       maximumPreviewSize: Number(this.maximumPreviewSize),
     }
-    this.quickview = CoveoHeadless.buildQuickview(engine, {options});
+    this.quickview = CoveoHeadless.buildQuickview(engine, { options });
```

Comment:
we can revert this prettier change. sidenote: we need to edit the rules because it is indeed annoying that it does this by default. 🙈